cluster/terraform_aks_cluster/config/test.tfvars.json

```diff
@@ -6,7 +6,17 @@
   "node_pools": {
     "applications": {
       "min_count": 2,
-      "max_count": 20
+      "max_count": 20,
+      "node_labels": {
+        "teacherservices.cloud/node_pool": "applications"
+      }
+    },
+    "apps1": {
```

Comment:
Can we use the new VMs in test as well? We may want to deploy apps with a lot of memory. And this would be consistent with production

Reply:
Added in latest push